frontend/src/contexts/HostContext/hostReducerMiddleware.js

```diff
@@ -104,13 +104,15 @@ async function getDeck({ selectedPacks }) {
   const apiURL = process.env.REACT_APP_API_URL;
   const queryString = selectedPacks.join(',');
   const query = `${apiURL}/deck/cards?packs=${queryString}`;
-  try {
-    const cardsRequest = await fetch(query);
+  const errorMessage = `Error fetching cards. Query: ${query}`;
+  const cardsRequest = await fetch(query);
+  if (cardsRequest.ok) {
     const cards = await cardsRequest.json();
-    return cards;
-  } catch {
-    throw new Error(`Error fetching cards. Query: ${query}`);
+    if (cards.white.length) {
+      return cards;
+    }
```

Comment:
This is fine for now, but I'm not sure the purpose of this. What if there's no black cards?